lib/puppet/functions/cache_data.rb

```diff
@@ -0,0 +1,43 @@
+require 'fileutils'
+require 'yaml'
+require 'etc'
+
+# @summary Retrieves data from a cache file, or creates it with supplied data if the file doesn't exist
```

Comment:
ooh, so `@summary` did work?  The puppet-strings style guide doesn't say to use it.
https://tickets.puppetlabs.com/browse/PDOC-263 also mentions this, but says it does fix the TOC.